cmd/kpod/run.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/libpod"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var runDescription = "Runs a command in a new container from the given image"
+
+var runCommand = cli.Command{
+	Name:        "run",
+	Usage:       "run a command in a new container",
+	Description: runDescription,
+	Flags:       createFlags,
+	Action:      runCmd,
+	ArgsUsage:   "IMAGE [COMMAND [ARG...]]",
+}
+
+func runCmd(c *cli.Context) error {
+	if err := validateFlags(c, createFlags); err != nil {
+		return err
+	}
+	runtime, err := libpod.NewRuntime()
+	if err != nil {
+		return errors.Wrapf(err, "error creating libpod runtime")
+	}
+
+	createConfig, err := parseCreateOpts(c, runtime)
+	if err != nil {
+		return err
+	}
+
+	createImage := runtime.NewImage(createConfig.image)
+
+	if !createImage.HasImageLocal() {
```

Comment:
A lot of this is common with Create - can we share it? Have a "createContainer" function that parses the opts, pulls the image, makes a new container, ctr.Create(), and returns the new container?